forgerock-guice-core/pom.xml

```diff
@@ -29,12 +30,28 @@
     <name>ForgeRock Guice Core</name>
     <description>ForgeRock Guice Core Library</description>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
Please let me know why you added maven-surefire-plugin.

Reply:
Thank you for the review.
I deleted this part because it is unnecessary.